src/types/actions/dialog-action.ts

```diff
@@ -26,6 +26,13 @@ export interface DialogSubmitAction {
   action_ts: string;
   token: string;
   response_url: string;
+
+  // exists for enterprise installs
+  is_enterprise_install?: boolean;
```

Comment:
Haven't confirmed if the payload is actually getting these two keys yet in enterprise install (and that team is null)